installers/redhat-docker/radiasoft-download.sh

```diff
@@ -32,19 +32,32 @@ Then re-run this command
         perl -pi -e 's{(?<=^SELINUX=).*}{disabled}' /etc/selinux/config
         install_err 'Disabled selinux. You need to "vagrant reload", then re-run this installer'
     fi
-    if type dnf >& /dev/null; then
+    declare o=rhel
+    if [[ $install_os_release_id == fedora ]]; then
         dnf -y -q install dnf-plugins-core
+        o=fedora
+    fi
+    dnf -q config-manager \
+        --add-repo \
+        https://download.docker.com/linux/"$o"/docker-ce.repo
+    dnf -y -q install docker-ce
+
+    if install_os_is_fedora; then
+        install_yum_install dnf-plugins-core
         dnf -q config-manager \
             --add-repo \
             https://download.docker.com/linux/fedora/docker-ce.repo
-        dnf -y -q install docker-ce
-    else
+    elif install_os_is_centos_7; then
         yum-config-manager \
             --add-repo https://download.docker.com/linux/centos/docker-ce.repo
         yum makecache fast
-        yum -y -q install yum-plugin-ovl
-        yum -y -q install docker-ce
+        install_yum_install yum-plugin-ovl
+    else
```

Comment:
Probably good at this point to ask is_rhel and fail in the else case.

Reply:
736a7be